kufar/books/main.go

```diff
@@ -318,12 +325,23 @@ func (d Database) LogoutUser(w http.ResponseWriter, r *http.Request) {
 
 //CreateBook ...
 func (d Database) CreateBook(w http.ResponseWriter, r *http.Request) {
+	var userID int
+
 	userID, err := checkSession(w, r)
 	if err != nil {
 		http.Error(w, http.StatusText(http.StatusUnauthorized)+": unauthorized access", http.StatusUnauthorized)
 		return
 	}
 
+	var valid bool
+
+	if userID == cookieCase {
```

Comment:
make my `checkAuth` func to return the userID, so even though the credentials are validated, only the owner with that bookID can update the book